mscore/shortcut.cpp

```diff
@@ -3246,11 +3246,11 @@ Shortcut Shortcut::_sc[] = {
       {
          MsWidget::SCORE_TAB,
          STATE_NORMAL | STATE_NOTE_ENTRY,
-         "add-parenthesis",
```

Comment:
What puzzlerd me though is that there has some compat code been added to read114() and read206(), as if it had been possible to use `[]` in 1.x and 2.x, see e.g. https://github.com/musescore/MuseScore/commit/48f2f0897f535085559c12aa2ceec1df1d634555#diff-9c93d8fdd487e712679e845aba7e30b6R245